internal/librarian/release_please_lite.go

```diff
@@ -39,9 +39,14 @@ func GetConventionalCommitsSinceLastRelease(repo gitrepo.Repository, library *co
 }
 
 // GetConventionalCommitsSinceLastGeneration returns all conventional commits for
-// the given library since the last generation.
+// all API paths in given library since the last generation.
 func GetConventionalCommitsSinceLastGeneration(repo gitrepo.Repository, library *config.LibraryState) ([]*conventionalcommits.ConventionalCommit, error) {
-	commits, err := repo.GetCommitsForPathsSinceCommit(library.SourceRoots, library.LastGeneratedCommit)
+	apiPaths := make([]string, 0)
+	for _, oneAPI := range library.APIs {
+		apiPaths = append(apiPaths, oneAPI.Path)
+	}
+
+	commits, err := repo.GetCommitsForPathsSinceCommit(apiPaths, library.LastGeneratedCommit)
```

Comment:
What would you expect to happen if LastGeneratedCommit is empty? (I'm expecting that to be common when onboarding a new library.) I think in that case we should just not bother looking for anything.

Reply:
It's a good point.

How about I file a new issue and fix it in another PR?